.env.example

```
@@ -51,6 +51,8 @@ SLACK_WEBHOOK_URL=👻
 
 # Metabase analytics
 METABASE_PORT=5000
+METABASE_API_KEY=👻
+METABASE_URL_EXT=👻
```

This is actually a genuine environmental variable and not a secret, so I think we're fine to just do the following here - 
```suggestion
METABASE_URL_EXT=http://localhost:${METABASE_PORT}
```

---

(Sorry if that wasn't clear from above!)

This just means that we have one fewer secret to manage which is always a bonus 👍